package.json

```diff
@@ -17,25 +17,20 @@
   "scripts": {
     "linter:fix": "prettier --write '{,!(node_modules|.git|storybook-static)/**/}*.{js,json,jsx,md}'",
     "linter:check": "eslint stories/ && eslint src/",
-    "storybook": "start-storybook -p 3006 -s ./public",
-    "release": "node build.js",
-    "release:test": "node build.js test",
+    "serve": "start-storybook -p 3006 -s ./public",
```

Comment:
make sure the infrastructure we're using is also updated with this command (shouldn't be used but double checking can't hurt).

Reply:
✔️